broker/src/main/java/io/camunda/zeebe/broker/system/partitions/impl/AsyncSnapshotDirector.java

```diff
@@ -119,24 +118,16 @@ public void forceSnapshot() {
   @Override
   protected void handleFailure(final Exception failure) {
     LOG.error(
-        "Snapshot director '{}' failed in phase {} with: {}",
+        "Uncaught exception at snapshot director '{}' in phase {}: {}",
```

Comment:
May be add a bit more context to the log messag. Something like `No snapshot will be taken now. Will try to take a snapshot at the next snapshot period. Please report the issue if the error persists.`.